mllib/src/main/scala/org/apache/spark/ml/tree/treeModels.scala

```diff
@@ -95,11 +95,6 @@ private[ml] trait TreeEnsembleModel[M <: DecisionTreeModel] {
   /** Trees in this ensemble. Warning: These have null parent Estimators. */
   def trees: Array[M]
 
-  /**
-   * Number of trees in ensemble
-   */
-  val getNumTrees: Int = trees.length
```

Comment:
Is there a reason this was moved? It seems unrelated to the cleanup described in the title.